yoti-sdk-api/src/main/java/com/yoti/api/client/HumanProfile.java

```diff
@@ -24,18 +24,18 @@
     /**
      * Equal to ${given_names} + " " + ${family_name}.
```

Comment:
I'm not sure this statement is correct.
Sorry for being pedantic, but I think this is quite important.  We need to nail these definitions, client devs will take them as gospel, and this kind of attribute is right at the core of what we do.

Reply:
This is what I assumed originally, realised it was wrong after Quirino pointed it out above, I'll change it.